### PR TITLE
As a user I can add a module to a contact #54

### LINKED ISSUE #54: As a user I can add a module to a contact
... so that I can keep track of which modules a person is taking and his availability

### PR DESCRIPTION
**Pull request by Kannan171**
Fixes #54 

Modules can be added by add or edit command using "-m ".

### CODE REVIEW

File: src/main/java/syncsquad/teamsync/storage/JsonAdaptedPerson.java [hunk 1]
```diff
@@ -36,11 +38,14 @@ class JsonAdaptedPerson {
     @JsonCreator
     public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
             @JsonProperty("email") String email, @JsonProperty("address") String address,
-            @JsonProperty("tags") List<JsonAdaptedTag> tags) {
+            @JsonProperty("modules") List<JsonAdaptedModule> modules, @JsonProperty("tags") List<JsonAdaptedTag> tags) {
```
Comment (xplus2g4) [inline src/main/java/syncsquad/teamsync/storage/JsonAdaptedPerson.java]: add a newline here

File: src/test/java/syncsquad/teamsync/logic/parser/AddressBookParserTest.java [hunk 2]
```diff
@@ -57,6 +57,7 @@ public void parseCommand_delete() throws Exception {
     public void parseCommand_edit() throws Exception {
         Person person = new PersonBuilder().build();
         EditPersonDescriptor descriptor = new EditPersonDescriptorBuilder(person).build();
+        System.out.println(PersonUtil.getEditPersonDescriptorDetails(descriptor));
```
Comment (xplus2g4) [inline src/test/java/syncsquad/teamsync/logic/parser/AddressBookParserTest.java]: Delete unnecessary print statements

File: src/test/java/syncsquad/teamsync/model/person/ModuleTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        assertFalse(Module.isValidModule("")); // empty string
+        assertFalse(Module.isValidModule(" ")); // spaces only
+        assertFalse(Module.isValidModule("comp")); // alphabet only
+        assertFalse(Module.isValidModule("2103")); // numbers only
+        assertFalse(Module.isValidModule("compsci2101")); // more than 4 letter department code
+        assertFalse(Module.isValidModule("c2101")); // less than 2 letter department code
+        assertFalse(Module.isValidModule("cs201")); // less than 3 letter numeric code
+        assertFalse(Module.isValidModule("cs21043")); // more than 4 letter numeric code
+        assertFalse(Module.isValidModule("cs2103tr")); // more than 1 alphabet suffix
+        assertFalse(Module.isValidModule("cs 2103t")); // spaces between code
```
Comment (xplus2g4) [inline src/test/java/syncsquad/teamsync/model/person/ModuleTest.java:36]: Nice catch of corner cases

**Review by xplus2g4 — APPROVED** [review]
LGTM! Nice work!
